docs/building-on-flow/nodes/node-operation/index.md

```diff
@@ -20,15 +20,32 @@ Flow multirole architecture makes it more scalable and provides several node typ
 
 The different types of nodes are described [here](./node-roles.mdx). As node operator, you can choose to run any of the different types of node that best fits your needs.
 
-### Observer node
-The observer node is one of the easiest node to spin up and can be run by Dapp developers who need the latest block data available locally e.g. a wallet application that needs to track the latest block ID and height.
+## Light client a.k.a. Observer node
```

Comment:
see a preview of this page [here](https://docs-git-vishal-nodechanges-onflow.vercel.app/next/building-on-flow/nodes/node-operation)